hmmlearn/base.py

```diff
@@ -102,6 +102,9 @@ def __init__(self, n_components=1, startprob=None, transmat=None,
         self.transmat_prior = transmat_prior
         self.algorithm = algorithm
         self.random_state = random_state
+        self.logprob_ = None
+        self.n_iterations_performed = None
+        self.logprob_delta = None
```

Comment:
I'm not sure we want this as an extra attribute, because the deltas can be trivially computed from `len(self.logprob_)`.